ome_zarr/writer.py

```diff
@@ -13,18 +13,75 @@
 
 LOGGER = logging.getLogger("ome_zarr.writer")
 
+KNOWN_AXES = {"x": "space", "y": "space", "z": "space", "c": "channel", "t": "time"}
 
-def _validate_axes_names(
-    ndim: int, axes: Union[str, List[str]] = None, fmt: Format = CurrentFormat()
-) -> Union[None, List[str]]:
-    """Returns validated list of axes names or raise exception if invalid"""
+
+def _axes_to_dicts(
```

Comment:
The growth and the complexity of the different methods here make me wonder whether:

1. we want to create an object to handle the creation/validation/conversion
2. we want to split the format specificity into `AxesV03`, `AxesV04`

Reply:
Created an Axes class in 9f75ae3